docs/firmware/state.schema.json

```
@@ -64,6 +64,40 @@
       },
       "required": ["v", "ts"]
     },
+    "tmp": {
+      "description": "Temperature reading in celcius",
+      "type": "object",
+      "properties": {
+        "v": {
+          "description": "Temperature reading from external sensor",
+          "type": "number",
+          "minimum": 1
```

```suggestion
          "minimum": 1
```
Temperature can be negative.